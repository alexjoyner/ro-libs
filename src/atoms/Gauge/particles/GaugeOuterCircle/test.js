import React from 'react';
import { shallow } from 'enzyme';
import { GaugeOuterCircle } from './';

describe('GaugeOuterCircle component', () => {
  let component;
  beforeEach(() => {
    component = shallow(<GaugeOuterCircle />);
  });
  it('Should render without exploding', () => {
    expect(component).toBeDefined();
  });
  it('Should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
