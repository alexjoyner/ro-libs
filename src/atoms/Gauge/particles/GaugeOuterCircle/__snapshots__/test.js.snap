// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GaugeOuterCircle component Should match snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "GaugeOuterCircle-sc-14kr5ii-0",
        "isStatic": true,
        "rules": Array [
          "stroke:#34495e;stroke-width:1;stroke-dasharray:389.557;stroke-dashoffset:389.557;",
        ],
      },
      "displayName": "GaugeOuterCircle",
      "render": [Function],
      "styledComponentId": "GaugeOuterCircle-sc-14kr5ii-0",
      "target": "circle",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
